### ISSUE TITLE
General QA & Organization 

### ISSUE THREAD

**Issue by Brook-A-CS** [CSAllenISD/2023-ISP-Bug-Tracker#22]
We have a lot of small issues we need to address. First, the new files I created each have their own folders, but I'd like to move them into the folders that were originally created based on which page their used on. I'd also like to rename many of the files and functions, the main ones being 'Add issue' and 'Change Issue' having page appended to the end of each of them. Let me know what you think of organizing them this way and let me know if you have any suggestions. 

- [ ] Reorganize project file tree
- [ ] More accurate file and func names

**Comment by Aidan-V** [comment]
For now we can hold on this as we need to work on other functionality cases, however, for quality I recently updated all grammar issues.

**Comment by Rafid-A** [comment]
I agree that we need to hold off on this for the time being as we still need to sort our the most basic of our program, however I suggest that we can work on something like this over the spring break.

**Comment by Rafid-A** [comment]
I recently took the time to clear up redundant code and fix CSS files on a website. This was an important task as it greatly improved the website's overall performance and user experience. By removing unnecessary code, I was able to streamline the website and ensure that it loaded quickly and efficiently. Additionally, fixing CSS files helped to ensure consistency in the design and layout of the website, making it more visually appealing to users.

**Comment by Rafid-A** [comment]
This is practically done on all branches and will be done on main branch this week
